src/EditorFeatures/VisualBasicTest/Classification/SemanticClassifierTests.vb

```diff
@@ -640,6 +640,5 @@ End Operator"
 
             Await TestInClassAsync(code)
         End Function
-
     End Class
```

Comment:
>Class [](start = 8, length = 5)

Let's revert this file (only change is a new line). Same for `AdvancedOptionPageControl.xaml`.

Reply:
Code doesn't get cleaned up... unless people clean it up :'(

Reply:
Those didn't seem like cleaning up (ie. not better).
Up to you...